compiler/src/dotty/tools/dotc/transform/Pickler.scala

```diff
@@ -55,35 +92,20 @@ class Pickler extends Phase {
     }
     {
       val pickler = new TastyPickler(cls)
-      if (ctx.settings.YtestPickler.value) {
+      if ctx.settings.YtestPickler.value then
         beforePickling(cls) = tree.show
         picklers(cls) = pickler
-      }
       val treePkl = pickler.treePkl
       treePkl.pickle(tree :: Nil)
       treePkl.compactify()
```

Comment:
If I remember correctly, `compactify` is the part of pickling I saw showing up in profiling, it should  be possible to move it in the parallel task too.

Reply:
Yes I wanted to try that one next.